test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -3331,6 +3331,82 @@ def test_checkpoint_abort_preemption(self):
 
         self.server.expect(JOB, {'job_state': 'R', 'substate': 42}, id=jid2)
 
+    @requirements(num_moms=2)
+    def test_checkpoint_restart(self):
+        """
+        Test to make sure that when a preempted and checkpointed multi-node
+        job restarts, execjob_begin cgroups hook gets called on both mother
+        superior and sister moms.
+        """
+        # Skip test if number of mom provided is not equal to two
+        if len(self.moms) != 2:
```

Comment:
Do we need this check with the @requirements decorator?

Reply:
No we do not need this check. Removed.